demo/src/server.ts

```diff
@@ -0,0 +1,67 @@
+import {https} from 'follow-redirects';
+import getStream from 'get-stream';
+import {IncomingMessage} from 'http';
+import Koa from 'koa';
+import koaCompress from 'koa-compress';
+import url from 'url';
+
+import {HTMLRewriter, jsRewrite, PackageJson, parsePackageName} from './html-rewriter';
+import {proxy} from './proxy';
+
+export class RawService {
+  app = new Koa();
+  private port = process.env.PORT || 8080;
```

Comment:
would probably make port a constructor param and use process.env.PORT when it's instantiated